beamer/tests/contracts/test_request_manager.py

```diff
@@ -347,168 +328,172 @@ def _get_claim_termination(_claim_id: ClaimId) -> Termination:
+        request_manager.challengeClaim(claim_id, sender=claimer, value=claim_stake + 1)
 
 
 def test_withdraw_nonexistent_claim(request_manager):
     """Test withdrawing a non-existent claim"""
-    with brownie.reverts("claimId not valid"):
-        request_manager.withdraw(1234, {"from": alloc_accounts(1)[0]})
+    with ape.reverts("claimId not valid"):
+        request_manager.withdraw(1234, sender=alloc_accounts(1)[0])
 
 
 def test_claim_nonexistent_request(request_manager):
     """Test claiming a non-existent request"""
-    (claimer,) = alloc_whitelisted_accounts(1, {request_manager})
-    with brownie.reverts("requestId not valid"):
-        request_manager.claimRequest(1234, FILL_ID, {"from": claimer})
+    (claimer,) = alloc_whitelisted_accounts(1, [request_manager])
+    with ape.reverts("requestId not valid"):
+        request_manager.claimRequest(b"1234", FILL_ID, sender=claimer)
 
 
-def test_claim_request_extension(request_manager, token, claim_stake):
+def test_claim_request_extension(request_manager, token, claim_stake, accounts):
     """
     Test that claiming is allowed around expiry
     and will revert after validUntil + claimRequestExtension
     """
     (requester,) = alloc_accounts(1)
-    (claimer,) = alloc_whitelisted_accounts(1, {request_manager})
-    token.mint(requester, 1, {"from": requester})
+    (claimer,) = alloc_whitelisted_accounts(1, [request_manager])
+    token.mint(requester, 1, sender=requester)
     request_id = make_request(request_manager, token, requester, requester, 1)
 
     valid_until = request_manager.requests(request_id)[RM_R_FIELD_VALID_UNTIL]
     claim_request_extension = request_manager.claimRequestExtension()
     # test that request expiration does not prevent claiming
     timestamps = [valid_until - 1, valid_until, valid_until + claim_request_extension - 1]
 
-    for timestamp in timestamps:
-        chain.sleep(timestamp - chain.time())
-        request_manager.claimRequest(request_id, FILL_ID, {"from": claimer, "value": claim_stake})
+    with accounts.use_sender(claimer):
+        for timestamp in timestamps:
+            chain.provider.set_timestamp(timestamp - chain.pending_timestamp)
+            request_manager.claimRequest(request_id, FILL_ID, value=claim_stake)
 
-    # validUntil + claimRequestExtension
-    chain.sleep(1)
-    with brownie.reverts("Request cannot be claimed anymore"):
-        request_manager.claimRequest(request_id, FILL_ID, {"from": claimer, "value": claim_stake})
+        # validUntil + claimRequestExtension
+        chain.provider.set_timestamp(valid_until + claim_request_extension)
+        with ape.reverts("Request cannot be claimed anymore"):
+            request_manager.claimRequest(request_id, FILL_ID, value=claim_stake)
 
 
-def test_withdraw_without_challenge(request_manager, token, claim_stake, claim_period):
+def test_withdraw_without_challenge(
+    request_manager, token, claim_stake, claim_period, web3, accounts
```

Comment:
Is `web3` a fixture now?

Reply:
it is hard to reach :D. `ape.chain.provider.web3`